.pre-commit-config.yaml

```diff
@@ -9,5 +9,5 @@ repos:
       - id: copyright_checker
         name: copyright_checker
         entry: python ./hack/gen-copyright.py
-        language: system
+        language: python
```

Comment:
Revert this change as it's covered in #374. Could you approve and merge that first?